distribution/docs/pom.xml

```diff
@@ -119,6 +121,42 @@
     </profiles>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.codehaus.gmaven</groupId>
+                <artifactId>groovy-maven-plugin</artifactId>
+                <version>2.1</version>
+                <inherited>false</inherited>
+                <executions>
+                    <execution>
+                        <id>aggregate-deps</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>execute</goal>
+                        </goals>
+                        <configuration>
+                            <source>
```

Comment:
🖊 Consider extracting into a stand-alone script file then referencing here with `<scriptpath>...` that should make it easier to not have labels in the execution flow

Reply:
I tried this but it breaks: `java.io.FileNotFoundException: /projects/ddf/${highest-basedir} -> [Help 1]`